daisy_workflows/build-publish/windows/windows-10-21h2-ent-x64-uefi.publish.json

```diff
@@ -0,0 +1,46 @@
+{{/*
+  Template to publish Windows server UEFI images.
+  By default this template is setup to publish to the 'gce-image-builder'
+  project, the 'environment' variable can be used to publish to 'test', 'prod',
+  or 'staging'.
+  DeleteAfter is set to 180 days for all environments other than prod where no
+  time period is set.
+*/}}
+{
+  "Name": "windows-10-21h2-ent-x64-uefi",
+  {{$work_project := printf "%q" "gce-image-builder" -}}
+  {{$endpoint := `"https://www.googleapis.com/compute/alpha/projects/"` -}}
```

Comment:
Any idea why the alpha endpoint is required? (I see it in some other publish.json files, I'm just curious)

Reply:
I believe at one point we were using methods that weren't supported in v1, although it's escaping me which ones they were. Alpha may not be entirely necessary anymore - I'd have to look into that.

Reply:
we still use alpha because of the rollout policy